### PR TITLE
add base

### PR DESCRIPTION
**Pull request by anhpham2403**

### CODE REVIEW

File: app/src/main/java/com/framgia/moviedb/screen/BaseViewModel.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.framgia.moviedb.screen;
+
+/**
+ * Created by anh on 19/09/2017.
+ */
+
+/**
+ * BaseView
+ * @Param <T> this is type paremeter
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moviedb/screen/BaseViewModel.java]: WARNING: Unknown tag 'Param'.

File: app/src/main/java/com/framgia/moviedb/screen/BaseViewModel.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ */
+public interface BaseViewModel<T extends BasePresenter> {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moviedb/screen/BaseViewModel.java]: WARNING: Type Javadoc comment is missing an @param <T> tag.

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -4,4 +4,6 @@
     <color name="color_primary_dark">#303F9F</color>
     <color name="color_accent">#FF4081</color>
     <color name="color_white">#FFFFFF</color>
+    <color name="color_black">#000000</color>
```
Comment (DoanVanToan) [inline app/src/main/res/values/colors.xml]: WARNING: [UnusedResources] The resource `R.color.color_black` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -4,4 +4,6 @@
...
+    <color name="materier_gray_300">#E0E0E0</color>
```
Comment (DoanVanToan) [inline app/src/main/res/values/colors.xml]: WARNING: [UnusedResources] The resource `R.color.materier_gray_300` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="card_movie_radius">0dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.card_movie_radius` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    <dimen name="card_movie_margin">10dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.card_movie_margin` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+    <dimen name="text_title">18sp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.text_title` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+    <dimen name="movie_date_release">15sp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.movie_date_release` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+    <dimen name="tab_indicator_height">5dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.tab_indicator_height` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+    <dimen name="image_poster_height">170dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values/dimens.xml]: WARNING: [UnusedResources] The resource `R.dimen.image_poster_height` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">MovieDB</string>
+    <string name="description_image_poster">poster</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.description_image_poster` appears to be unused
 Unused resources make applications larger and slow down builds.

File: app/src/main/res/values/dimens.xml (hunk 9 above)
Comment (anhpham2403) [reply app/src/main/res/values/dimens.xml]: Em sẽ dùng pull sau

File: app/src/main/res/values/dimens.xml (hunk 10 above)
Comment (anhpham2403) [reply app/src/main/res/values/dimens.xml]: Em sẽ dùng pull sau

File: app/src/main/res/values/strings.xml (hunk 11 above)
Comment (anhpham2403) [reply app/src/main/res/values/strings.xml]: Em sẽ dùng pull sau

File: app/src/main/res/values/dimens.xml (hunk 6 above)
Comment (anhpham2403) [reply app/src/main/res/values/dimens.xml]: Em sẽ dùng pull sau

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (anhpham2403) [reply app/src/main/res/values/colors.xml]: Em sẽ dùng pull sau

File: app/src/main/res/values/colors.xml (hunk 3 above)
Comment (anhpham2403) [reply app/src/main/res/values/colors.xml]: Em sẽ dùng pull sau

**Pull request merged**